src/containers/OrderReviewPage.jsx

```diff
@@ -274,9 +280,13 @@ class OrderReviewPage extends React.Component {
                         <CheckoutSummaryList data={orderData} isLoading={false} />
                       </div>
                       <div className="checkout-submit-container" style={{ display: 'block' }}>
-                        <button className="ep-btn primary wide btn-cmd-submit-order" type="button" onClick={() => { this.completeOrder(); }}>
+                        <button className="ep-btn primary wide btn-cmd-submit-order" disabled={!isValid} type="button" onClick={() => { this.completeOrder(); }}>
                           {intl.get('complete-purchase')}
                         </button>
+                        <br />
+                        <button className="ep-btn primary wide btn-cmd-submit-order" type="button" onClick={() => { this.goToCheckOut(); }}>
```

Comment:
Can you please move this "edit" button below where the address & payment information is displayed? Within the above div with class name `order-options-container`